djmoney/admin.py

```diff
@@ -11,10 +12,12 @@
 def setup_admin_integration():
     original_display_for_field = admin_utils.display_for_field
 
-    def display_for_field(value, field, empty):
+    def display_for_field(value, field, empty, avoid_link=False):
```

Comment:
To avoid having to test against a specific Django version you could also do:
```
def display_for_field(value, field, *args, **kwargs):
    if isinstance(field, MoneyField):
        return str(value)
    return original_display_for_field(value, field, *args, **kwargs)
```

This would also make the code more resistant to other changes to the signature in the future.